sfge/src/main/java/com/salesforce/config/UserFacingMessages.java

```diff
@@ -1,11 +1,32 @@
 package com.salesforce.config;
 
 /**
- * Contains error message constants that will be displayed to users. TODO: move all other
- * user-facing messages here.
+ * Contains error message constants that will be displayed to users.
+ * TODO: move all other user-facing messages here.
+ * TODO: Reorganize these messages into a layered system like in {@link com.salesforce.apex.jorje.ASTConstants}
```

Comment:
I've been handling moving constants as and when I work on related things so that we don't have to wait to prioritize and pick it up. If you feel you could cover a few other groups, feel free to modify them.